### ISSUE TITLE
CNN Model Integration & Performance Evaluation

### ISSUE THREAD

**Issue by DhanushGWU1995** [DhanushGWU1995/Final-Project-Group2#12]
Load and contruct CNN model
Customize final layers for the classification task
Freeze/unfreeze layers for fine-tuning
Train the model and monitor performance
Plot training vs validation loss and accuracy
Compare results with other advanced models
